test/Website/Program.cs

```diff
@@ -21,14 +21,13 @@ public static IHostBuilder CreateHostBuilder(string[] args)
                         builder
                             .AddConsul(
                                 "appsettings.json",
-                                CancellationTokenSource.Token,
                                 options =>
                                 {
                                     options.ConsulConfigurationOptions =
                                         cco => { cco.Address = new Uri("http://consul:8500"); };
                                     options.Optional = true;
+                                    options.PollWaitTime = TimeSpan.FromSeconds(5);
```

Comment:
In ConsulConfigurationSource the PollWaitTime is 5 minutes. Is this different intentionally?

Reply:
Is this not just to demonstrate you _can_ adjust the poll wait time?

Reply:
Ah right, makes sense

Reply:
This is also just a test project. I think I set it lower just to test out what it did when the config was missing. It's not really too important.